Mathlib/FieldTheory/Adjoin.lean

```diff
@@ -412,24 +412,16 @@ theorem adjoin_subset_adjoin_iff {F' : Type*} [Field F'] [Algebra F' E] {S S' :
 theorem adjoin_adjoin_left (T : Set E) :
     (adjoin (adjoin F S) T).restrictScalars _ = adjoin F (S ∪ T) := by
   rw [SetLike.ext'_iff]
-  change (↑(adjoin (adjoin F S) T) : Set E) = _
-  apply Set.eq_of_subset_of_subset <;> rw [adjoin_subset_adjoin_iff] <;> constructor
+  change (adjoin (adjoin F S) T : Set E) = _
+  apply subset_antisymm <;> rw [adjoin_subset_adjoin_iff] <;> constructor
   · rintro _ ⟨⟨x, hx⟩, rfl⟩; exact adjoin.mono _ _ _ Set.subset_union_left hx
   · exact subset_adjoin_of_subset_right _ _ Set.subset_union_right
--- Porting note: orginal proof times out
-  · rintro x ⟨f, rfl⟩
-    refine Subfield.subset_closure ?_
-    left
-    exact ⟨f, rfl⟩
--- Porting note: orginal proof times out
-  · refine Set.union_subset (fun x hx => Subfield.subset_closure ?_)
-      (fun x hx => Subfield.subset_closure ?_)
-    · left
-      refine ⟨⟨x, Subfield.subset_closure ?_⟩, rfl⟩
-      right
-      exact hx
-    · right
-      exact hx
+  -- Porting note: orginal proof times out
+  · exact Set.range_subset_iff.mpr fun f ↦ Subfield.subset_closure (.inl ⟨f, rfl⟩)
+  -- Porting note: orginal proof times out
```

Comment:
These porting notes seem to be decidedly unhelpful without knowing what the original proof was...
Can we just remove them?

Reply:
Probably they mean that there exist some shorter proof that times out, so they have more than zero information content ... I'll take a look at the original proof in mathlib3.

Reply:
Oh, this is actually a proof I golfed. Will remove the porting notes then!